openshift/Jenkinsfile

```diff
@@ -60,6 +60,8 @@ pipeline {
                     sh "PROJ_TARGET=${projProd} CPU_REQUEST=1000m CPU_LIMIT=2000m ./openshift/scripts/oc_provision_backup_postgres_cronjob.sh prod apply"
                     sh "PROJ_TARGET=${projProd} BACKUP_VOLUME_SIZE=3Gi ./openshift/scripts/oc_provision_backup_mariadb.sh prod apply"
                     sh "PROJ_TARGET=${projProd} ./openshift/scripts/oc_provision_backup_mariadb_cronjob.sh prod apply"
+                    // Configure 
+                    sh "PROJ_TARGET=${projProd} CERTBOT_STAGING=false DRYRUN=false CERTBOT_SERVER=\"https://www.entrust.net/acme/api/v1/directory/98-YMBC-M0BH\" ./openshift/scripts/oc_provision_certbot_cronjob.sh prod apply"
```

Comment:
what is the `98-YMBC-blahblah` about? Is that something specific to us (and if so, should we be making it a config map/secret variable?

Reply:
As far as I know it's not a secret - I'll ask.